test/Garnet.test/GarnetBitmapTests.cs

```diff
@@ -2302,7 +2302,7 @@ public void BitmapOperationInvalidOption()
             }
             catch (Exception ex)
             {
-                Assert.AreEqual("syntax error", ex.Message);
+                Assert.AreEqual("ERR unknown command", ex.Message);
```

Comment:
Sure, I'll make the attempt though it will complicate parsing a bit now that subcommands are handled at the parsing layer.

I will note that the old test was incorrect (Redis's actual response is `-ERR syntax error`, not `-syntax error`), and that  Redis does not document the text of errors (and it can change between minor versions). See the [RESP2 documentation on errors](https://github.com/redis/redis-specifications/blob/master/protocol/RESP2.md#resp-errors).

Reply:
> Sure, I'll make the attempt though it will complicate parsing a bit now that "subcommands" are handled at the parsing layer.
> 
> I will note that the old test was incorrect (Redis's actual response is `-ERR syntax error`, not `-syntax error`), and that Redis does not document the text of errors (and it can change between minor versions). See the [RESP2 documentation on errors](https://github.com/redis/redis-specifications/blob/master/protocol/RESP2.md#resp-errors).

just for your reference,

https://github.com/microsoft/garnet/blob/f864b0b3e573e8da847413f197a20615bb625e0e/libs/server/Resp/CmdStrings.cs#L120